src/components/Title.js

```diff
@@ -3,7 +3,7 @@ import { colors } from '../config';
 
 export const Title = styled.Text`
   color: ${colors.primary};
-  font-weight: bold;
+  font-family: titillium-web-bold;
```

Comment:
please add `font-weight: 600;` here, or is it bold anyway?

Reply:
it's already bold thanks to  titillium-web-bold

Reply:
so we can remove `font-weight: 600;` from everywhere, where we ware using `titillium-web-bold`?

Reply:
yes, done.